workbench-core/base/src/aws/helpers/dynamoDB/dynamoDBService.ts

```diff
@@ -450,18 +450,36 @@ export default class DynamoDBService {
   }
 
   public transactEdit(params?: {
-    addPutRequest?: Record<string, unknown>[];
+    addPutRequests?: {
```

Comment:
Are these all of the options available or is there a full schema somewhere we could reference so we know we've got all available properties exposed?

Reply:
https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_Put.html

Reply:
With one missing attribute, `ReturnValuesOnConditionCheckFailure`,  but can be added when needed